File: Beecrowd/Beginner/1037 - Interval/C/1037_Interval.c
#include <stdio.h>

int main() {
    double number;
    scanf("%lf", &number);

    if (number >= 0 && number <= 25) printf("Intervalo [0,25]\n");
    else if (number > 25 && number <= 50) printf("Intervalo (25,50]\n");
    else if (number > 50 && number <= 75) printf("Intervalo (50,75]\n");
    else if (number > 75 && number <= 100) printf("Intervalo (75,100]\n");
    else printf("Fora de intervalo\n");

    return 0;
}

File: Beecrowd/Beginner/1000 - 1099/1037 - Interval/Python/1037_Interval.py
number = float(input())

if 0 <= number <= 25:
    print("Intervalo [0,25]")
elif 25 < number <= 50:
    print("Intervalo (25,50]")
elif 50 < number <= 75:
    print("Intervalo (50,75]")
elif 75 < number <= 100:
    print("Intervalo (75,100]")
else:
    print("Fora de intervalo")